members/G000555.yaml

```diff
@@ -65,13 +65,41 @@ contact_form:
             Pastor: Pastor
             Rev.: Rev.
             Other: Other
-    - click_on:
-        - name: subject
-          selector: "#issue-13"
-          value: Government Reform
+    - picker:
+        - name: Topic
+          value: $TOPIC
```

Comment:
This will require you to setup a topic map

Reply:
there's one already set up, albeit outdated. I'm updating it on artemis right now